### ISSUE TITLE
Where is the Concept obtained from？

### ISSUE THREAD

**Issue by janicewym** [imJiawen/KEFDA#3]
Hi, 
    After reading your paper, I am very interested in your idea. I am doing some experiments with Wikidata and I want to know how did you get the Concept of the entity (the file "ent2cnpt.txt") from Wikidata?
…………………………………………………………………………………………………………………………………………………………………………
你好， 我想问一下，你的实验中用到的Concept是从Wikidata中的哪个字段抽取出来的？

**Comment by imJiawen** [comment]
Hi,
Thank you for your interest.
Basically, I use query `SELECT $WDid WHERE {wd:<entity_uri> wdt:P31 ?WDid}` to find an entity’s corresponding concepts. 

For more details, please refer to:
[1] [Wikidata Query Service](https://query.wikidata.org/)
[2] [Pywikibot](https://pypi.org/project/pywikibot/)

If needed, I will upload my code later. 
But this part is not complicated, so I recommend building it by yourself to meet your needs.

**Comment by janicewym** [comment]
Thank you!&nbsp;


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "imJiawen/KEFDA"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月26日(星期天) 上午10:22
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [imJiawen/KEFDA] Where is the Concept obtained from？ (#3)





 
Hi,
 Thank you for your interest.
 Basically, I use query SELECT $WDid WHERE {wd:<entity_uri&gt; wdt:P31 ?WDid} to find an entity’s corresponding concepts.
 
For more details, please refer to:
 [1] Wikidata Query Service
 [2] Pywikibot
 
If needed, I will upload my code later.
 But this part is not complicated, so I recommend building it by yourself to meet your needs.

**Comment by yijunL** [comment]
> Hi, Thank you for your interest. Basically, I use query `SELECT $WDid WHERE {wd:<entity_uri> wdt:P31 ?WDid}` to find an entity’s corresponding concepts.
> 
> For more details, please refer to: [1] [Wikidata Query Service](https://query.wikidata.org/) [2] [Pywikibot](https://pypi.org/project/pywikibot/)
> 
> If needed, I will upload my code later. But this part is not complicated, so I recommend building it by yourself to meet your needs.

您好，请问可以提供一下构建ontology的代码吗，非常感谢，我的邮箱是yijunleo@163.com
